packages/editable-html/src/index.jsx

```diff
@@ -47,7 +47,7 @@ export default class EditableHtml extends React.Component {
     const v = htmlToValue(props.markup);
     const current = htmlToValue(this.props.markup);
 
-    if (!v.equals(current)) {
+    if (v.equals && !v.equals(current)) {
```

Comment:
Why do you need this? Is there a point when this function does not exist?

Reply:
yes, we need this in the case that` serializer.deserialize(html)` does not happen